### PR TITLE
Add "Playground" (aka json-browser) to website

### PR DESCRIPTION
**Pull request by ibgreen**
- This PR adds the `json-browser` as a website example. 

Screen Shot 2019-09-10 at 1 25 12 PM

### CODE REVIEW

**Comment by Pessimistress** [comment]
- I wouldn't add it as an embedded demo because of the visual layout. We can add a link to the demo page which launches a new page.
- I wouldn't call it "JSON layers", it can be confusing.

**Comment by ibgreen** [comment]
> I wouldn't add it as an embedded demo because of the visual layout. We can add a link to the demo page which launches a new page.
> I wouldn't call it "JSON layers", it can be confusing.

Both good comments, makes sense. I will dig into it, but might need some help/pointers with how to set it up as a link within the current page setup.

I like having it on the `https://deck.gl/json` url. Not sure if the current website publishing process can handle that...

**Comment by ibgreen** [comment]
> I wouldn't call it "JSON layers", it can be confusing.

Let's find a name...

- JSON Playground?

But maybe we should not even emphasize the `JSON` part - it seems to cause no end of confusion, as people associate it with other formats that happen to be JSON based...

**Comment by Pessimistress** [comment]
> might need some help/pointers with how to set it up as a link within the current page setup.

Check the "showcase" page - thumbnail is added to the markdown and the side bar link is added to the table of contents with an `external` flag.

**Comment by Pessimistress** [comment]
I think "playground" is a good name. It will also be helpful to add links to the source code and json module documentation.

**Comment by ibgreen** [comment]
OK, let's go with just "Playground".

Though one thing I was looking forward to was having the "Playground" built as part of the website release process, to ensure it stays working with the latest framework changes. 

My understanding is that he showcases are for static examples that are separately built and deployed and not actively maintained, so making it into such an example still keeps this as a very separate thing, with all the fragility that comes with that...

**Comment by ibgreen** [comment]
> Check the "showcase" page - thumbnail is added to the markdown and the side bar link is added to the table of contents with an external flag.

How about this?

Screen Shot 2019-09-10 at 2 51 17 PM

File: website/contents/pages.js [hunk 1]
```diff
@@ -159,6 +159,16 @@ export const examplePages = generatePath([
         }
       }
     ]
+  },
+  {
+    name: 'Interactive Layers',
```
Comment (Pessimistress) [inline website/contents/pages.js]: Declarative? Live-Editable?

**Comment by ibgreen** [comment]
Comments addressed
- Code and json-examples reorganized and collected inside `examples/playground`.
- Renamed to `Declarative Layers` for now, feel free to change.

**Pull request merged**